src/pages/registration/contributors_tab/Authors.tsx

```diff
@@ -32,6 +33,15 @@ const Authors = ({ push, replace }: AuthorsProps) => {
   const [unverifiedAuthor, setUnverifiedAuthor] = useState<UnverifiedContributor | null>(null);
   const orderedAuthors = contributors.map((contributor, index) => ({ ...contributor, sequence: index + 1 }));
 
+  const [width, setWidth] = useState(window.innerWidth);
+  const isMobile = width < lightTheme.breakpoints.values.md;
+  const handleResize = () => setWidth(window.innerWidth);
+
+  useEffect(() => {
```

Comment:
Nå vil vel handleResize opprettes på nytt for hver render, så unmount har ikke nødvendigvis samme funksjon som på mount
```suggestion

  useEffect(() => {
    const handleResize = () => setWidth(window.innerWidth);
```

Reply:
åh sant.